packages/gatsby-plugin-typedoc/src/gatsby-node.ts

```diff
@@ -102,8 +137,8 @@ export function createSchemaCustomization({
       }
 
       type TypedocComment @dontInfer {
-        text: String
-        shortText: String
+        text: TypedocCommentValue!
```

Comment:
you want the `@link` directive as well, which set the gql resolver to resolve the src value as an id and return the node instead. e.g. it does this:

```js
resolve: (obj) => getNode(src)
```